app/views/dradis/plugins/csv/upload/new.html.erb

```diff
@@ -0,0 +1,81 @@
+<% content_for :title, 'CSV Upload Mapping' %>
+
+<% content_for :breadcrumbs do %>
+  <nav>
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item">
+        <%= link_to 'Upload Manager', main_app.project_upload_manager_path(current_project) %>
+      </li>
+      <li class="breadcrumb-item active">CSV Upload Mapping</li>
+    </ol>
+  </nav>
+<% end %>
+
+<div class="content-container">
+  <%= "Uploaded file: #{@attachment.filename}" %>
+
+  <table class="table table-striped mb-0"
+    data-behavior="dradis-datatable"
+    data-item-name="csv_item"
+    data-local-storage-key="project.ce.<%= dom_id(current_project) %>.csv_mapping_datatable"
+  >
+    <thead>
+      <tr>
+        <th class="no-sort">Unique Indentifier</th>
+        <th>Column Header From File</th>
+        <th>Type</th>
+        <th>Field in Dradis</th>
+        <%# if RTP && RTP has fields defined %>
```

Comment:
why are we using ERB comments here and elsewhere in this file?

Reply:
These will be removed for the other PRs